harness/determined/deploy/local/cli.py

```diff
@@ -30,6 +30,7 @@ def handle_cluster_up(args: argparse.Namespace) -> None:
         gpu=args.gpu,
         autorestart=(not args.no_autorestart),
         auto_work_dir=args.auto_work_dir,
+        enterprise_edition=args.enterprise_edition,
```

Comment:
How do you see this working for `det deploy gcp`, etc.?

Reply:
det deploy gcp and aws already have the `--enterprise-edition` option. Local is the only one that doesn't have it and previously depended on whether you were `det deploy`ing from EE or OSS repositories

Reply:
🙄 Thanks. I'd gotten myself confused reading `--help`s and had missed that. LGTM.